docs/proposals/20201020-management-cluster-operator.md

```diff
@@ -98,14 +102,13 @@ management cluster based on a declarative API.
 
 ### Non-Goals/Future Work
 
-- Deprecate the clusterctl CLI
-- Manage cluster templates using the operator
-- Implement an operator driven version of `clusterctl move`
-- Manage cert-manager using the operator
+- Deprecate the clusterctl CLI.
+- Manage cluster templates using the operator.
+- Implement an operator driven version of `clusterctl move`.
+- Manage cert-manager using the operator.
 - Support multiple installations of the same provider within a management
-  cluster in light of [issue
-  3042](https://github.com/kubernetes-sigs/cluster-api/issues/3042) and [issue
-  3354](https://github.com/kubernetes-sigs/cluster-api/issues/3354)
+  cluster in light of [issue 3042] and [issue 3354].
+- Support multiple template processors.
```

Comment:
I would add a non-goal to support the installation of v1alpha3 providers using the operator